pkg/apis/v1beta1/nodeclaim_status.go

```diff
@@ -22,12 +22,13 @@ import (
 )
 
 const (
-	ConditionTypeLaunched    = "Launched"
-	ConditionTypeRegistered  = "Registered"
-	ConditionTypeInitialized = "Initialized"
-	ConditionTypeEmpty       = "Empty"
-	ConditionTypeDrifted     = "Drifted"
-	ConditionTypeTerminating = "Terminating"
+	ConditionTypeLaunched               = "Launched"
+	ConditionTypeRegistered             = "Registered"
+	ConditionTypeInitialized            = "Initialized"
+	ConditionTypeEmpty                  = "Empty"
+	ConditionTypeDrifted                = "Drifted"
+	ConditionTypeTerminating            = "Terminating"
+	ConditionTypeInconsistentStateFound = "InconsistentStateFound"
```

Comment:
Considering how the status conditions are setup with operatorpkg, we actually probably want to model this as a positive if true. In other words, this should resolve to "NodeClaim is healthy if this status condition is true", where this current status condition says "this nodeclaim is unhealthy if this is true".